### DISCUSSION TITLE
WSSD can't discover any computers in network

### DISCUSSION THREAD

**Discussion by pindos74** [christgau/wsdd#166] [Q&A] [answered]
Hello! I installed WSSD on Linux Mint 21.1 and Ubuntu 22.04. These systems are newly installed on VM Ware. I try to use WSSD with default settings. Windows 7,10,11 can see Linux hosts. But Linux doesn't see any hosts. Then I start "wssd -D", as it written in manual it is discovery mode. The situation is the same. No Windows or Linux hosts in Network of Ubuntu or Linux Mint. Firewall are disabled on all systems for test purposes. I don't understand what's the problem is? Can WSSD make Windows hosts visible in network? By the way, I can connect to these hosts with smb://hostname/share_name command.

**Comment by christgau** [comment, marked as answer]
> Hello! I installed WSSD on Linux Mint 21.1 and Ubuntu 22.04. These systems are newly installed on VM Ware. I try to use WSSD with default settings. Windows 7,10,11 can see Linux hosts. 

Then everything is fine. With the default settings, i.e. if nothing else specified, only discovery by Windows is enabled. 


> But Linux doesn't see any hosts. 

This is expected. See above. And in addition, where do you expect to "see any host" on Linux? That statement above is rather unprecise. 

> Then I start "wssd -D", as it written in manual it is discovery mode. The situation is the same. No Windows or Linux hosts in Network of Ubuntu or Linux Mint. 

Ah. Now, it becomes more clear. AFAIK, there is no integration into any file manager or desktop environment. To see detected hosts on the command line, you have to specify `-v` at least once. Alternatively, you can use the command line interface. 

> I don't understand what's the problem is? Can WSSD make Windows hosts visible in network? 

No. That's not was wsdd is (currently) for. 

> By the way, I can connect to these hosts with smb://hostname/share_name command.

Yes, because service detection, which is what the WSD protocol was made for, is - from a pure technical point of view - not related to service availability.

**Reply by pindos74** [reply]
Thanks! Now, it is clear. I believed that wssd will help to see hosts in linux network environment via GUI. :)
